src/software/geom/vector.h

```diff
@@ -191,6 +191,20 @@ class Vector final
      */
     Vector &operator=(const Vector &other);
 
+    /**
+     * Returns true if this vector is to the right of the given vector.
```

Comment:
should say +X instead of right of and -X for left of, if that's what you mean

Reply:
left and right are in terms of the relative orientation here. But ya, maybe the naming should be updated so its not confused with relative x/y position.

Reply:
updated the documentation of vector.h to make the functions a little clearer hopefully